GX-TimeFrequency/src/minimax_tau.F90

```diff
@@ -2921,10 +2925,24 @@ subroutine get_points_weights_tau(grid_size, e_range, ac_we, ierr)
        allocate(aw%aw_erange_matrix(2*grid_size, bup+1))
        call set_aw_array(grid_size, aw)
```

Comment:
I know this already exists but if one's going to use a type/class, then initialisation etc should be done with type-bound procedures. Perhaps consider changing this